### DISCUSSION TITLE
Hide signature from completion menu

### DISCUSSION THREAD

**Discussion by arkrost** [hrsh7th/nvim-cmp#1786] [Q&A] [answered]
Hi! How can I hide signatures from the completion menu?
<img width="1460" alt="Screenshot 2023-12-25 at 18 58 25" src="https://github.com/hrsh7th/nvim-cmp/assets/1239844/5b53f4b7-dd12-4c83-837d-2fbf3b2fc8f1">

**Comment by fitrh** [comment, marked as answer]
How does your cmp configuration look like? I think you can do something like this

```lua
require('cmp').setup({
  formatting = {
    format = function(_, item)
      item.menu = ""
      return item
    end,
  },
})
```
